pdr_backend/lake/table.py

```diff
@@ -60,38 +59,3 @@ def _append_to_db(
         PersistentDataStore(self.base_path).insert_to_table(data, table_name)
         n_new = data.shape[0]
         print(f"  Just saved df with {n_new} df rows to the database of {table_name}")
-
-    def get_pds_last_record(self) -> Optional[pl.DataFrame]:
-        """
-        Get the last record from the persistent data store
-
-        @returns
-            pl.DataFrame
-        """
-
-        query = f"SELECT * FROM {self.table_name} ORDER BY timestamp DESC LIMIT 1"
```

Comment:
Although this only returns 1 record, I agree that this should be removed because it's not being used anywhere.